flight/Modules/Stabilization/stabilization.c

```diff
@@ -941,13 +976,25 @@ static void SettingsUpdatedCb(UAVObjEvent * ev)
 		// The dT has some jitter iteration to iteration that we don't want to
 		// make thie result unpredictable.  Still, it's nicer to specify the constant
 		// based on a time (in ms) rather than a fixed multiplier.  The error between
-		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this
-		// calculation
+		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this calculation
+		//fakeDt now only used for vbar_decay, for gyro filter now a real, but filtered, dT is used
 		const float fakeDt = 0.0025f;
-		if(settings.GyroTau < 0.0001f)
-			gyro_alpha = 0;   // not trusting this to resolve to 0
+
+		//TODO: if the infrastructure for alarms will be more advanced in the future,
+		//      we should assert a alarm here when GyroCutoff > 90% of F_nyquist and LPF gets disabled
+
+		if (settings.GyroCutoff < 1.0f) {	// disable LPF when GyroCutoff < 1.0 Hz
+			gyro_filter_enabled = GYRO_LPF_DISABLED;
+			settings.GyroCutoff = 0;
+		}
+		else if ( (dT > 0) && (settings.GyroCutoff > (0.45f / dT)) ) {	// disable LPF when GyroCutoff > 90% of F_nyquist (= Fs/2 = 1/(2*dT)); using here not the filtered dT, because this is at the start to low
```

Comment:
As requested the filtered dT is not calculated continuously, but only for the first 1000 "runs" of the task.

So i can check here if the calculations already has finished, otherwise this check won't be done. Additional when the filtered dT calculation is finished, there will also be the check if cutoff frequency is to high.

Is this OK?